saml-core/src/main/java/org/keycloak/saml/processing/core/saml/v2/util/AssertionUtil.java

```diff
@@ -560,11 +571,11 @@ public static ResponseType decryptAssertion(ResponseType responseType, PrivateKe
         SAMLParser parser = new SAMLParser();
 
         JAXPValidationUtil.checkSchemaValidation(decryptedDocumentElement);
-        AssertionType assertion = (AssertionType) parser.parse(StaxParserUtil.getXMLEventReader(DocumentUtil
+        AssertionType assertion = (AssertionType) parser.parse(parser.createEventReader(DocumentUtil
```

Comment:
This is a change from https://github.com/keycloak/keycloak/pull/3886 I need, otherwise assertions with line breaks fails.